simulation/src/test/resources/report.md

```diff
@@ -1,40 +1,40 @@
-                              all_nodes_500[UNLIMITED_ROUND_ROBIN].txt:	success=50.0%	client_mean=PT0.6S         	server_cpu=PT20M          	client_received=2000/2000	server_resps=2000	codes={200=1000, 500=1000}
+                all_nodes_500[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=82.6%	client_mean=PT3.077742714S 	server_cpu=PT47M28.2S     	client_received=2000/2000	server_resps=4747	codes={200=1651, 500=349}
+                    all_nodes_500[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=71.6%	client_mean=PT2.40517074S  	server_cpu=PT55M6S        	client_received=2000/2000	server_resps=5510	codes={200=1431, 500=569}
+                              all_nodes_500[UNLIMITED_ROUND_ROBIN].txt:	success=70.6%	client_mean=PT2.406374783S 	server_cpu=PT55M15S       	client_received=2000/2000	server_resps=5525	codes={200=1412, 500=588}
                    black_hole[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=30.0%	client_mean=PT0.6S         	server_cpu=PT6M           	client_received=600/2000	server_resps=600	codes={200=600}
                        black_hole[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=90.0%	client_mean=PT0.600011117S 	server_cpu=PT17M59.4S     	client_received=1799/2000	server_resps=1799	codes={200=1799}
                                  black_hole[UNLIMITED_ROUND_ROBIN].txt:	success=68.3%	client_mean=PT0.6S         	server_cpu=PT13M39.6S     	client_received=1366/2000	server_resps=1366	codes={200=1366}
              drastic_slowdown[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT0.075961S    	server_cpu=PT5M3.844S     	client_received=4000/4000	server_resps=4000	codes={200=4000}
                  drastic_slowdown[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT2.060500083S 	server_cpu=PT2H17M22.000333313S	client_received=4000/4000	server_resps=4000	codes={200=4000}
                            drastic_slowdown[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT9.158069333S 	server_cpu=PT10H10M32.277333313S	client_received=4000/4000	server_resps=4000	codes={200=4000}
         fast_500s_then_revert[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT0.080925333S 	server_cpu=PT5M3.47S      	client_received=3750/3750	server_resps=3750	codes={200=3750}
-            fast_500s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=76.0%	client_mean=PT0.055333733S 	server_cpu=PT3M27.501499711S	client_received=3750/3750	server_resps=3750	codes={200=2849, 500=901}
-                      fast_500s_then_revert[UNLIMITED_ROUND_ROBIN].txt:	success=76.0%	client_mean=PT0.055333733S 	server_cpu=PT3M27.501499711S	client_received=3750/3750	server_resps=3750	codes={200=2849, 500=901}
+            fast_500s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=99.2%	client_mean=PT0.173513629S 	server_cpu=PT4M53.194832836S	client_received=3750/3750	server_resps=5217	codes={200=3721, 500=29}
```

Comment:
dramatic improvement here too. I know multipass had a campaign of getting all their heavy users to switch to ROUNB_ROBIN, so I think this is not pointless.

https://github.com/palantir/dialogue/blob/dfox/retry-5xx/simulation/src/test/resources/report.md#fast_500s_then_revertconcurrency_limiter_round_robin